docs/api/product_variants.rst

```diff
@@ -0,0 +1,756 @@
+--------------------------------
+
+You can request detailed product variant information by executing the following request:
+
+Definition
+^^^^^^^^^^
+
+.. code-block:: text
+
+    GET /api/v1/products/{productId}/variants/{id}
+
++---------------+----------------+----------------------------------------------------------+
+| Parameter     | Parameter type | Description                                              |
++===============+================+==========================================================+
+| Authorization | header         | Token received during authentication                     |
++---------------+----------------+----------------------------------------------------------+
+| id            | url attribute  | Id of requested resource                                 |
++---------------+----------------+----------------------------------------------------------+
+| productId     | url attribute  | Id of product for which the variants should be displayed |
++---------------+----------------+----------------------------------------------------------+
+
+Example
+.......
+
+.. code-block:: bash
+
+    $ curl http://sylius.dev/api/v1/products/1/variants/1 \
+        -H "Authorization: Bearer MWExMWM0NzE1NmUyZDgyZDJiMjEzMmFlMjQ4MzgwMmE4ZTkxYzM0YjdlN2U2YzliNDIyMTk1ZDhlNDYxYWE4Ng" \
+        -H "Accept: application/json"
+
+Example Response
+~~~~~~~~~~~~~~~~
+
+.. code-block:: text
+
+    STATUS: 200 OK
+
+.. code-block:: json
+
+    {
+        "id": 1,
+        "code": "319bf720-e91d-36f5-aa8e-8c674e3861fb-variant-0",
+        "option_values": [
+            {
+                "name": "Mug type",
+                "code": "mug_type_medium"
+            }
+        ],
+        "position": 0,
+        "translations": {
+            "en_US": {
+                "locale": "en_US",
+                "id": 1,
+                "name": "blanditiis"
+            }
+        },
+        "on_hold": 0,
+        "on_hand": 1,
+        "tracked": false,
+        "channel_pricings": [
+            {
+                "id": 1,
+                "channel": {
+                    "id": 1,
+                    "code": "US_WEB",
+                    "name": "US Web Store",
+                    "hostname": "localhost:8000",
+                    "color": "Sienna",
+                    "created_at": "2017-01-27T11:09:49+0100",
+                    "updated_at": "2017-01-27T11:09:49+0100",
+                    "enabled": true,
+                    "tax_calculation_strategy": "order_items_based",
+                    "_links": {
+                        "self": {
+                            "href": "/api/v1/channels/1"
+                        }
+                    }
+                },
+                "price": 872
+            }
+        ],
+        "_links": {
+            "self": {
+                "href": "/api/v1/products/1/variants/1"
+            }
+	    }
+    }
+
+Creating a Product Variant
+--------------------------
+
```

Comment:
Some small explanation would be really nice. At least it is what @TheMadeleine told me 🐋